### PR TITLE
Reference pages for 'multiplayer' blocks

### PR DESCRIPTION
**Pull request by ganicke**
Add the reference pages for the 'multiplayer' extension APIs.

Closes https://github.com/microsoft/pxt-arcade/issues/5515

### CODE REVIEW

**Review by eanders-ms — COMMENTED** [review]
Looks great! Left a few comments/suggestions. Love the samples :)

File: libs/multiplayer/docs/reference/multiplayer/on-controller-event.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    . . f f e e e e f f f f f e f . 
+    . f b d f e e f b b f f f e f . 
+    . f d d f f f f d d b f f f f . 
+    . f f f f f f f f f f f f f . . 
+    `, SpriteKind.Player))
+mp.getPlayerSprite(mp.playerSelector(mp.PlayerNumber.One)).setBounceOnWall(true)
+mp.getPlayerSprite(mp.playerSelector(mp.PlayerNumber.One)).setVelocity(40, -30)
+```
+
+## See also #seealso
```
[the remark this answers is not in the document]
Comment (ganicke) [reply libs/multiplayer/docs/reference/multiplayer/on-controller-event.md:68]: Yes, as well as `## Example #example`. Thanks for the catch.

File: libs/multiplayer/docs/reference/multiplayer/multiplayer-state.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+In blocks, a sprite kind identifier is used to make a sprite kind item that can be assigned to a variable or used as a parameter:
+
+```block
+let lifeState = MultiplayerState.life
+```
+
+In blocks, the multiplayer state list lets you add your own custom state identifers to it. Let's say you wanted to give players 'gems' as rewards during a game. When you add `gems` to the state list, it also adds a new state constant called `gems` to your project code:
+
+```typescript-ignore
+namespace MultiplayerState {
```
[the remark this answers is not in the document]
Comment (ganicke) [reply libs/multiplayer/docs/reference/multiplayer/multiplayer-state.md:27]: Back-compat == regrets we must all live with.

**Review by eanders-ms — APPROVED** [review]
:shipit:

**Review by jwunderl — APPROVED** [review]

**Pull request merged**